content/_includes/server-config.md

```diff
@@ -273,15 +253,15 @@ keystore:
       engine: ""  # The path of the transit engine - e.g. "my-transit". If empty, defaults to: transit (Vault default)
       key: ""     # The key name that should be used to encrypt entries stored on the K/V engine.
     approle:    # AppRole credentials. See: https://www.vaultproject.io/docs/auth/approle.html
-      engine: ""  # The path of the AppRole engine - e.g. authenticate. If empty, defaults to: approle. (Vault default)
-      id: ""      # Your AppRole Role ID
-      secret: ""  # Your AppRole Secret ID
-      retry: 15s  # Duration until the server tries to re-authenticate after connection loss.
+      namespace: "" # Optional Vault namespace used just for authentication. A single "/" is an alias for the Vault root namespace.
```

Comment:
Idea:
```suggestion
      namespace: "" # Optional Vault namespace used only for authentication. For the Vault root namespace, use "/".
```
I'm not sure if it is significant that `/` is an alias, ignore if I have it wrong.

Reply:
👍